src/components/sds/Input/index.tsx

```diff
@@ -264,142 +265,154 @@ interface InputProps {
-  ...props
-}) => {
-  const handleCopy = () => {
-    if (!value) {
-      return;
-    }
+export const Input = React.forwardRef<TextInput, InputProps>(
+  (
+    {
+      fieldSize = "md",
+      label,
+      labelSuffix,
+      isLabelUppercase,
+      isError,
+      isPassword,
+      leftElement,
+      rightElement,
+      note,
+      error,
+      success,
+      copyButton,
+      endButton,
+      value = "",
+      onChangeText,
+      placeholder,
+      editable = true,
+      testID,
+      autoCorrect = true,
+      ...props
+    },
+    ref,
+  ) => {
+    const [showPassword] = useState(false);
 
-    Clipboard.setString(value);
-  };
+    const handleCopy = () => {
+      if (!value) {
+        return;
+      }
 
-  const getLabelSize = () => ({
-    xs: fieldSize === "sm",
-    sm: fieldSize === "md",
-    md: fieldSize === "lg",
-  });
+      Clipboard.setString(value);
+    };
 
-  const renderCopyButton = (position: "left" | "right") => (
-    <TouchableOpacity onPress={handleCopy}>
-      <SideElement position={position}>
-        <Text sm>{copyButton?.showLabel ? "Copy" : "📋"}</Text>
-      </SideElement>
-    </TouchableOpacity>
-  );
+    const getLabelSize = () => ({
+      xs: fieldSize === "sm",
+      sm: fieldSize === "md",
+      md: fieldSize === "lg",
+    });
 
-  return (
-    <Container $fieldSize={fieldSize}>
-      {label && (
-        <Text {...getLabelSize()} color={THEME.colors.text.secondary}>
-          {isLabelUppercase ? label.toString().toUpperCase() : label}
-          {labelSuffix && (
-            <Text {...getLabelSize()} color={THEME.colors.text.secondary}>
-              {" "}
-              {labelSuffix}
-            </Text>
-          )}
-        </Text>
-      )}
+    const renderCopyButton = (position: "left" | "right") => (
+      <TouchableOpacity onPress={handleCopy}>
+        <SideElement position={position}>
+          <Text sm>{copyButton?.showLabel ? "Copy" : "📋"}</Text>
+        </SideElement>
+      </TouchableOpacity>
+    );
 
-      <InputContainer
-        testID={testID ? `${testID}-container` : undefined}
-        $fieldSize={fieldSize}
-        $isError={isError || !!error}
-        $isDisabled={!editable}
-      >
-        {copyButton?.position === "left" && renderCopyButton("left")}
-        {leftElement && (
-          <SideElement marginSide="right">{leftElement}</SideElement>
+    return (
+      <Container $fieldSize={fieldSize}>
+        {label && (
+          <Text {...getLabelSize()} color={THEME.colors.text.secondary}>
+            {isLabelUppercase ? label.toString().toUpperCase() : label}
+            {labelSuffix && (
+              <Text {...getLabelSize()} color={THEME.colors.text.secondary}>
+                {" "}
+                {labelSuffix}
+              </Text>
+            )}
+          </Text>
         )}
 
-        <StyledTextInput
-          testID={testID}
+        <InputContainer
+          testID={testID ? `${testID}-container` : undefined}
```

Comment:
```suggestion
          testID={testID ? `${testID}-input-container` : undefined}
```

just to make it more unique

Reply:
made the id more unique ${testID}-input-container